addons/web_editor/static/src/components/media_dialog/media_dialog.js

```diff
@@ -37,7 +37,7 @@ export const TABS = {
 export class MediaDialog extends Component {
     setup() {
         this.size = 'xl';
-        this.contentClass = 'o_select_media_dialog';
+        this.contentClass = 'o_select_media_dialog h-100';
```

Comment:
Can't the `Dialog` `isFullscreen` property help us here? Isn't it making it full screen?

Reply:
Apparently not, it seems to only impact the header of the dialog
It seems to remove the cross to close the dialog and to add an arrow before the title, to "go back" 

![image](https://github.com/odoo/odoo/assets/102796037/5ec84767-eb08-45b8-a095-23fb6d130cb9)

Reply:
Ok so the property is to tell the dialog that it is (already) a full screen dialog and to adapt the header, and it does not tell the dialog to make it full screen, sad :|